tracer/src/Datadog.Trace.ClrProfiler.Native/calltarget_tokens.h

```diff
@@ -86,7 +86,8 @@ class CallTargetTokens
                                         mdToken* callTargetReturnToken, ILInstr** firstInstruction);
 
     HRESULT WriteBeginMethod(void* rewriterWrapperPtr, mdTypeRef integrationTypeRef, const TypeInfo* currentType,
-                             std::vector<FunctionMethodArgument>& methodArguments, ILInstr** instruction);
+                             std::vector<FunctionMethodArgument>& methodArguments, std::vector<USHORT>& argsToLoad,
```

Comment:
nit: Maybe the new vector can be passed as a `const`

Reply:
Done in c9cc188a9ed5c2b65f1a7d1600fa80431ff335e2